codecs/opus/src/test/java/org/restcomm/media/codec/opus/OpusCodecTest.java

```diff
@@ -70,26 +79,61 @@ public void tearDown() throws Exception {
      * Test of process method, for Encoder and Decoder.
      */
     @Test
-    public void testCodec() {
-    	OpusJni opus = new OpusJni();
-    	opus.setOpusObserverNative(this);
-    	opus.sayHelloNative();
+    public void testCodec() throws Exception {
+    	
+    	boolean testPassed = false;
+    	
+    	try {
```

Comment:
For safety, use try-with-resources for any closeable resource used (inputStream, outputStream).
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Done